ExoPlayerSampleApp/app/build.gradle

```diff
@@ -89,12 +89,12 @@ dependencies {
   compile files('libs/FWAdManager.jar')
   compile files('libs/OoyalaFreewheelSDK.jar')
   compile files('libs/OoyalaIMASDK.jar')
-  compile files('libs/ima-android-v330.jar')
+  compile files('libs/ima-android-v371.jar')
   compile files('libs/OoyalaSDK.jar')
   compile 'com.google.android.exoplayer:exoplayer:r1.5.7'
 
 // For development purposes only
-//  compile project(':ooyalaCoreSDK')
+//  compile project(':core')
```

Comment:
While importing sdk repository name is 'core', 'ooyalaCoreSDK' is not relevant. Thats the reason every time you enable this you have to change name 'ooyalaCoreSDK' to 'core' also along with uncommenting. Just made little easier.